streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -797,11 +797,11 @@ public void testThroughputMetrics() {
         final String entity = "entity";
         final String operation = "put";
 
-        final Sensor sensor1 = streamsMetrics.addThroughputSensor(scope, entity, operation, RecordingLevel.DEBUG);
```

Comment:
You can also remove the `@SuppressWarnings("deprecation")` at the top of the method.